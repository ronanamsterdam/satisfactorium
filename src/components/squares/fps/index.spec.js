import React from 'react'

import Fps from './index'

describe(`fps meter`, () => {
  test(`renders fps`, () => {
    expect(shallow(<Fps />)).toMatchSnapshot();
  })
})
