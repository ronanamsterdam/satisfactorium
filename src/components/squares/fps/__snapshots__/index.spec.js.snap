// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fps meter renders fps 1`] = `ShallowWrapper {}`;
